### PR TITLE
Backend only: add Polygon and xDai chains for DAO membership

### PR DESCRIPTION
**Pull request by mattcan**
This uses two new subgraphs to pull DAO memberships for Ethereum, xDai, and Polygon mainnets. The pulls are in parallel. The underlying data is structured the same way but seems to be missing some information, see https://github.com/HausDAO/daohaus-supergraph/issues/44.

These are backend only changes and so the UI is a little bit wonky. Since some DAOs will have no title, there is a default that follows the format of `Unknown ${chain} DAO`. This is pretty meaningless but more meaningful than showing a blank title. Frontend changes will be required to make this look better.

This is split from #581.

## Screenshots

*Before*:

Only Ethereum mainnet DAOs

643-before

*After*:

You can see that there are a number of DAOs without titles

643-altchain-default-title

### CODE REVIEW

File: packages/backend/src/lib/daoHausClient.ts [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
-import { getSdk } from './autogen/daohaus-sdk';
+import { getSdk, Sdk } from './autogen/daohaus-sdk';
 
-export const daoHausClient = getSdk(
-  new GraphQLClient(CONFIG.daoHausGraphqlURL),
-);
+export function clientFactory(chain: string): Sdk {
+  switch (chain) {
+    case 'polygon':
+      return getSdk(new GraphQLClient(CONFIG.daoHausPolygonGraphqlURL));
```
Comment (dan13ram) [inline packages/backend/src/lib/daoHausClient.ts:9]: I'd suggest caching this locally. Otherwise a new GraphQLClient is created on every api call.

**Review by dan13ram — CHANGES_REQUESTED** [review]
Everything looks good. Just requesting a minor change to increase performance

File: packages/backend/src/lib/daoHausClient.ts (hunk 1 above)
Comment (mattcan) [reply packages/backend/src/lib/daoHausClient.ts]: Good catch! I have updated that now.

**Comment by mattcan** [comment]
I have modified the way the resolver works to be less surprising and handle errors a bit better. Full detail in https://github.com/MetaFam/TheGame/pull/643/commits/bb600dbbed9fe0c46542c7185ea7bab35b303fa9

**Review by dan13ram — APPROVED** [review]
Awesome. Thank you, this is long overdue!

**Pull request merged**